chore: add Slack complex asyncapi fixture document with models and query params support (#1509)
asyncapi/generator: packages/templates/clients/websocket/python/README.md

6. Start example script that uses a client library generated by the test: `python example.py`

> By default this is testing against Hoppscotch echo service. You can modify `packages/templates/clients/websocket/python/example.py` and change line 7 to `from temp.snapshotTestResult.client_postman.client import PostmanEchoWebSocketClientClient` and line 21 `client = PostmanEchoWebSocketClientClient()` and run `python example.py` again. You will see example still works but agains different API. This is possible as both AsyncAPI documents have the same name of operation for sending messages: `sendEchoMessage` so each client generated has the same API.

## Client for Slack

> You run the example the same way as described above with 2 exceptions:

Reviewer: ```suggestion
To run the Slack Client example, follow the steps above but with 2 exceptions:
```

> - Instead of `example.py` call `example-slack.py`

Reviewer: ```suggestion
- Use `example-slack.py` instead of `example.py`.
```

- You need to pass environment variables with proper authorization details. Your command must look like this: `TICKET={provide secret info} APP_ID={provide id of the slack app} python example-slack.py`

> Instructions how to create the Slack app and also obtain authorization is described in details in the [Slack AsyncAPI document](../test/__fixtures__/asyncapi-slack-client.yml).

Reviewer: ```suggestion
Instructions on how to create the Slack app and also obtain authorization is described in details in the [Slack AsyncAPI document](../test/__fixtures__/asyncapi-slack-client.yml).
```
